core/licenses/jna-4.4.0.jar.sha1

```diff
@@ -1 +1 @@
-6edc9b4514969d768039acf43f04210b15658cd7
\ No newline at end of file
+d85e9ceee4d97e738524a231687ff28446376c97
```

Comment:
I would actually prefer the extension to permanently differentiate (like an OS package), but this kind of change is so infrequent maybe it's not worth it.  Maybe I should take it up on elastic/jna-build#1.